vng/int.go

```diff
@@ -18,18 +19,17 @@ func (p *Int64Encoder) Write(v int64) {
 	p.PrimitiveEncoder.Write(super.EncodeInt(v))
 }
 
-type Int64Decoder struct {
-	PrimitiveBuilder
-}
-
-func NewInt64Decoder(loc Segment, r io.ReaderAt) *Int64Decoder {
-	return &Int64Decoder{*NewPrimitiveBuilder(&Primitive{Typ: super.TypeInt64, Location: loc}, r)}
-}
-
-func (p *Int64Decoder) Next() (int64, error) {
-	zv, err := p.ReadBytes()
-	if err != nil {
-		return 0, err
+func ReadUint32Vector(loc Segment, r io.ReaderAt) ([]uint32, error) {
```

Comment:
Nit: I'm aware of the precedent leading to the appearance of "Vector" in this name but think it's confusing given that what "vector" means in the rest of the system, so maybe call this something like ReadUint32s or ReadZcodeUint32s?

Reply:
Calling it ReadUint32s since using intcomp for uint32 vectors in dynamics, nulls, unions, etc. is the first place I'm using the intcomp library so the zcode stuff will go soon.